post.html

```diff
@@ -0,0 +1,13 @@
+---
+layout: default
+title: "%title%"
+description: "%description%"
+url: "%permalink%"
+header-img: "%header_img%"
+---
+<script>
+window.postData = `%postData%`;
```

Comment:
instead of putting a string here, and parse it in the Vue, can't we juste put the JSON directly here?

Reply:
Technically yes but you'd have to work around liquid with interpolations (like on the news page) and I found using string templates much cleaner